python/pip_install/pip_repository_requirements.bzl.tmpl

```diff
@@ -8,6 +8,8 @@ from %%REQUIREMENTS_LOCK%%
 
 all_requirements = %%ALL_REQUIREMENTS%%
 
+all_requirement_groups = %%ALL_REQUIREMENT_GROUPS%%
```

Comment:
Can we avoid using a global variable here? Why do we need to expose this value to our users?

Reply:
If we're doing the groups dance correctly this should be totally transparent -- happy to make this private.